packages/angular/src/generators/directive/directive.spec.ts

```diff
@@ -0,0 +1,163 @@
+import { addProjectConfiguration, Tree } from '@nrwl/devkit';
+import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing-pre16';
```

Comment:
Is there a reason to use the `pre16` one?
```suggestion
import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
```